publisher-sdk/src/main/java/com/criteo/publisher/model/Config.java

```diff
@@ -58,6 +59,7 @@
     private static final String AD_TAG_DATA_MODE = "<html><body style='text-align:center; margin:0px; padding:0px; horizontal-align:center;'><script>%%adTagData%%</script></body></html>";
     private static final boolean CSM_ENABLED = true;
     private static final boolean LIVE_BIDDING_ENABLED = false;
+    private static final int LIVE_BIDDING_TIME_BUDGET = BuildConfig.networkTimeoutInMillis;
```

Comment:
I wouldn't put a default equal to the networktimeout as it's quite big. Instead, I would put 5_000ms which is the baseline that we intend to set.

Reply:
Ok